x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.test.tsx

```diff
@@ -233,6 +236,28 @@ describe('RuleActionsOverflow', () => {
   });
 
   describe('rules details export rule', () => {
+    test('should call export actions without the onSuccess to use the default prop', async () => {
```

Comment:
Nit: I'd make this message more descriptive to make it easier to understand that this is the success case, like `should call export actions and display toast when export option is clicked`. I think the onSuccess prop is more of an implementation detail in this case.

Reply:
Good point 👍  please check e9855fe , I just see that we test/expect the function calls with the component UI so I wanted to describe why I am expecting this result.

Reply:
> Thanks @WafaaNasr, changes LGTM 👍 The only thought for your consideration: maybe we should have an e2e test for that as well.
> 
> I didn't test the PR locally, only reviewed the changes.

Great, I will create new PR for adding the E2E tests for this bug-fix